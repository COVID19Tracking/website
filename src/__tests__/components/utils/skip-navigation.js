import React from 'react'
import renderer from 'react-test-renderer'
import SkipNavigation from '../../../components/utils/skip-navigation'

describe('Components : Utilities: Skip navigation link', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SkipNavigation />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
