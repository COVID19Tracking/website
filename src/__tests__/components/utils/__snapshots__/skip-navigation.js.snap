// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components : Utilities: Skip navigation link renders correctly 1`] = `
<a
  data-reach-skip-link=""
  data-reach-skip-nav-link=""
  href="#reach-skip-nav"
>
  Skip site navigation
</a>
`;
